src/fidesops/schemas/privacy_request.py

```diff
@@ -82,6 +82,7 @@ class PrivacyRequestResponse(BaseSchema):
     finished_processing_at: Optional[datetime]
     status: PrivacyRequestStatus
     external_id: Optional[str]
+    identity: Optional[Dict[str, str]]
```

Comment:
We could also have the type be the already defined `PrivacyRequestIdentity` schema for this too.  It makes the responses uniform, so if the identity doesn't exist, its value is null - 

```json
"identity": {
                "phone_number": null,
                "email": "customer-1@example.com"
            }
```

Reply:
I've added a comment here instead. My rationale is that we'd rather not expose the potential internals of our identity data structure on data egress, given this is an API response. ie. if we use this schema then we'd end up with
```
{"email": "email@example.com", "phone_number": null}
```
for responses where we have an `email` but no `phone_number`, and if we don't have the value it's better we don't expose it at all.